python_modules/dagster/dagster/_core/execution/context/system.py

```diff
@@ -1109,22 +1112,16 @@ def get_output_asset_keys(self) -> AbstractSet[AssetKey]:
             output_keys.add(asset_info.key)
         return output_keys
 
-    def has_asset_partitions_for_input(self, input_name: str) -> bool:
```

Comment:
What's the deprecation policy for these methods on StepExecutionContext? Are they considered public? If so i'll retain versions of these methods that just call their asset_key based counterparts after converting the input name to an asset_key

Reply:
Methods are only public if:
- They're on a publicly-exported class
- They have the `@public` decorator

Reply:
ok - these are public then. I'll add the method names back

Reply:
This one at least doesn't have the `@public` decorator, right?

Reply:
yeah, but StepExecutionContext is publicly exported class.

Reply:
wait... was the list of conditions an AND not an OR?

Reply:
ah oops yes, and AND

Reply:
ok i can revert that commit lol